Application/Transfer/Import/Standard/Mail/Service.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+namespace SPHERE\Application\Transfer\Import\Standard\Mail;
+
+use MOC\V\Component\Document\Component\Bridge\Repository\PhpExcel;
+use MOC\V\Component\Document\Document;
+use SPHERE\Application\People\Person\Person;
+use SPHERE\Application\Platform\Gatekeeper\Authorization\Account\Account;
+use SPHERE\Common\Frontend\Form\IFormInterface;
+use SPHERE\Common\Frontend\Layout\Repository\Panel;
+use SPHERE\Common\Frontend\Layout\Structure\Layout;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutColumn;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutGroup;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutRow;
+use SPHERE\Common\Frontend\Message\Repository\Danger;
+use SPHERE\Common\Frontend\Message\Repository\Success;
+use SPHERE\Common\Frontend\Message\Repository\Warning;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+/**
+ * Class Service
+ *
+ * @package SPHERE\Application\Transfer\Import\Standard\Mail
+ */
+class Service
+{
+    /**
+     * @param IFormInterface|null $Form
+     * @param UploadedFile $File
+     * @param null $Data
+     *
+     * @return IFormInterface|Danger|string
+     */
+    public function createMailsFromFile(IFormInterface $Form = null, UploadedFile $File = null, $Data = null)
+    {
+
+        /**
+         * Skip to Frontend
+         */
+        if (null === $File) {
+            return $Form;
+        }
+
+        if (!($tblType = \SPHERE\Application\Contact\Mail\Mail::useService()->getTypeById($Data['Type']))) {
+            $Form->setError('Data[Type]', 'Bitte geben Sie einen Typ an');
+            return $Form;
+        }
+        $isAccountAlias = isset($Data['IsAccountAlias']);
+        $isTest = isset($Data['IsTest']);
+
+        if (null !== $File) {
+            if ($File->getError()) {
+                $Form->setError('File', 'Fehler');
+            } else {
+
+                /**
+                 * Prepare
+                 */
+                $File = $File->move($File->getPath(), $File->getFilename() . '.' . $File->getClientOriginalExtension());
+                /**
+                 * Read
+                 */
+                /** @var PhpExcel $Document */
+                $Document = Document::getDocument($File->getPathname());
+
+                $X = $Document->getSheetColumnCount();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $X. Configured minimum length is 2.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)